src/main/scala/com/gu/fezziwig/CirceScroogeMacros.scala

```diff
@@ -94,10 +94,10 @@ private class CirceScroogeMacrosImpl(val c: blackbox.Context) {
           *   `scala.reflect.internal.Types$TypeError: value <none> is not a member of com.gu.fezziwig.FezziwigTests`
           */
         val decoderCopy = c.untypecheck(implicitDecoder)
-        val errMsg = s"Unable to find ${name.toString}"
+
         val accDecodeParam =
           q"""cursor.downField(${name.toString}).success
-            .map(x => $decoderCopy.accumulating(x)).getOrElse(_root_.cats.data.Validated.invalidNel(_root_.io.circe.DecodingFailure($errMsg, cursor.history)))"""
+            .map(x => $decoderCopy.accumulating(x)).getOrElse(_root_.cats.data.Validated.invalidNel(_root_.io.circe.DecodingFailure("Unable to find: " + ${name.toString}, cursor.history)))"""
```

Comment:
interesting, i tried to do this initially but it failed on an error like "don't know how to unquote".

can't exactly remember what my syntax was, but it appears that substitutions can't go into a quoted string _within_ a quasiquote??